src/Event.php

```diff
@@ -170,9 +169,19 @@ public function __construct()
 
     /**
      * Gets the UUID of this event.
+     *
+     * @return string|EventId
      */
-    public function getId(): string
+    public function getId(bool $returnAsString = true, bool $throwDeprecation = true)
```

Comment:
Why adding that second parameter? Calling `getId(true, false)` would NOT throw a deprecation but break in the next major. IMHO calling  `getId(false)->__toString()` obtains the same but avoiding the risk of silent future breakage.

Reply:
>Why adding that second parameter?

I wrongly assumed that it was needed to make some tests work, but you're right in saying that `getId(false)->__toString()` is more than enough to not trigger any deprecation in the unit tests and thus it's useless